Makefile

```diff
@@ -3,17 +3,20 @@
 PROJECT = probfit
 CYTHON ?= cython
 
+PYX_FILES := $(wildcard probfit/*.pyx)
+
+.PHONY: help clean build test coverage doc doc-show code-analysis flake8 pylint
+
 help:
 	@echo ''
 	@echo ' probfit available make targets:'
 	@echo ''
 	@echo '     help             Print this help message (the default)'
 	@echo ''
 	@echo '     clean            Remove generated files'
-	@echo '     build            Build inplace'
+	@echo '     build            Build Cython files'
```

Comment:
Oh, yes.
`python setup.py build_ext --inplace` does run Cython, build shared libraries, create pyc files or __pycache__ directories.
I had read "Build Cython files" as just running Cython, i.e. creating the C files.
So +1 to change the description back or to something more precise.

But overall it doesn't matter much ... put what you think best.